### PR TITLE
Added support for more fields

### PR DESCRIPTION
**Pull request by themason45**
- Django phonenumber_field
- Duration Field

### CODE REVIEW

File: graphene_django/converter.py [hunk 1]
```diff
@@ -56,8 +56,10 @@ def convert_django_field(field, registry=None):
     raise Exception(
         "Don't know how to convert the Django field %s (%s)" %
         (field, field.__class__))
-
-
+    
+from phonenumber_field.formfields import PhoneNumberField
```
Comment (yen223) [inline graphene_django/converter.py]: phonenumber_field is a third-party dependency

**Comment by nlhkh** [comment]
any progress on this?

**Comment by nickhudkins** [comment]
@syrusakbary Any opposition to getting this merged in?

**Comment by spockNinja** [comment]
`DurationField` is now in master, registered to the graphene `Float` type.

If you need to use a 3rd party model field like django-phonenumber-field, it is possible for you to register it yourself. Just be sure to put it at the top of your root `schema.py` file.

```python
from graphene_django.converter import convert_django_field

@convert_django_field.register(CustomModelField, registry=None):
    return SomeGrapheneType()
```

**Pull request closed without merging**